includes/admin/feedzy-rss-feeds-import.php

```diff
@@ -1952,4 +1965,31 @@ public function allow_iframe_tag_item_content( $tags, $context ) {
 		}
 		return $tags;
 	}
+
+	/**
+	 * Get post data using meta key and value.
+	 *
+	 * @param string $post_type Post type Default post.
+	 * @param string $key Meta Key.
+	 * @param string $value Meta value.
+	 * @param string $compare Compare operator.
+	 * @return mixed
+	 */
+	public function is_duplicate_post( $post_type = 'post', $key = '', $value = '', $compare = '=' ) {
+		if ( empty( $key ) || empty( $value ) ) {
+			return false;
+		}
+		// Check post exists OR Not.
+		$data = get_posts(
+			array(
+				'post_type' => $post_type,
+				'meta_key' => $key,
+				'meta_value' => $value,
+				'meta_compare' => $compare,
```

Comment:
please don't use unbounded queries since it present problems as the site scales, use posts_per_page=100 or lower.

Reply:
We have added 80 posts per import.